flink-connectors/flink-jdbc/src/test/java/org/apache/flink/api/java/io/jdbc/JDBCTableSourceITCase.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.java.io.jdbc;
+
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.EnvironmentSettings;
+import org.apache.flink.table.api.Table;
+import org.apache.flink.table.api.java.StreamTableEnvironment;
+import org.apache.flink.table.runtime.utils.StreamITCase;
+import org.apache.flink.types.Row;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * IT case for {@link JDBCTableSource}.
+ */
+public class JDBCTableSourceITCase extends JDBCTestBase {
+
+	private static final StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
+	private static final EnvironmentSettings bsSettings = EnvironmentSettings.newInstance().useBlinkPlanner().inStreamingMode().build();
+	private static final StreamTableEnvironment tEnv = StreamTableEnvironment.create(env, bsSettings);
+
+	static final String TABLE_SOURCE_SQL = "CREATE TABLE books (" +
+		" id int, " +
+		" title varchar, " +
+		" author varchar, " +
+		" price double, " +
+		" qty int " +
+		") with (" +
+		" 'connector.type' = 'jdbc', " +
+		" 'connector.url' = 'jdbc:derby:memory:ebookshop', " +
+		" 'connector.table' = 'books', " +
+		" 'connector.driver' = 'org.apache.derby.jdbc.EmbeddedDriver' " +
+		")";
+
+	static {
```

Comment:
Please use `@BeforeClass` instead of static code block.

Reply:
Thanks, Done.